### PR TITLE
Cmake depends on curl (close #381)

### LINKED ISSUE #381: cmake depends on curl
See https://github.com/tseemann/homebrew-bioinformatics-linux/issues/1

### PR DESCRIPTION
**Pull request by iggyvolz**
close #381.

### CODE REVIEW

File: Library/Formula/cmake.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Cmake < Formula
   option "without-docs", "Don't build man pages"
 
   depends_on :python => :build if OS.mac? && MacOS.version <= :snow_leopard && build.with?("docs")
+  depends_on "curl"
```
Comment (sjackman) [inline Library/Formula/cmake.rb]: curl is provided by default on Mac OS, so this dependency should not apply to Mac OS. Linuxbrew aims to maintain compatibility with Homebrew and not change the behaviour of Homebrew when Linuxbrew is run on a Mac. This line should be
```ruby
depends_on "curl" unless OS.mac?
```

Comment (iggyvolz) [inline Library/Formula/cmake.rb]: I knew that, sorry!

**Comment by sjackman** [comment]
To be consistent the commit message should start with `cmake:` rather than `Cmake`,  note the case  and colon.

**Comment by sjackman** [comment]
Please also include "for Linuxbrew" in the first line of the commit message.

**Comment by sjackman** [comment]
Thanks, Iggy.

**Pull request closed without merging**